frontend/packages/dev-console/src/components/taskruns/list-page/TaskRunsHeader.tsx

```diff
@@ -10,33 +10,39 @@ const TaskRunsHeader = () => {
       props: { className: tableColumnClasses[0] },
     },
     {
-      title: 'Namespace',
```

Comment:
Can you really remove Namespace? I think if you go to the Admin tab page and do all projects, you'll be missing this column, no?

Reply:
@andrewballantyne Thank you for pointing this out. I noticed in the pipelines list page that the namespace column shows up when the user is not in the context of a project. I have incorporated that in my PR and have added a GIF in PR description